aspnetcore/fundamentals/middleware/index.md

```diff
@@ -57,6 +59,27 @@ When a delegate doesn't pass a request to the next delegate, it's called *short-
 >
 > <xref:Microsoft.AspNetCore.Http.HttpResponse.HasStarted%2A> is a useful hint to indicate if headers have been sent or the body has been written to.
 
+### Short-circuit after routing
```

Comment:
This is a routing feature, not a middleware feature. Let's move it to the routing doc.
https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-8.0&branch=pr-en-us-29110#configure-endpoint-metadata
https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-8.0&branch=pr-en-us-29110#performance-guidance-for-routing